### DISCUSSION TITLE
Is anyone else inundated with Chromium tabs only searching for one product?

### DISCUSSION THREAD

**Discussion by occasional-it-guy** [jef/streetmerchant#1143] [Help]
### Description

So I'm running Streetmerchant on Mac and I've noticed that I'm overwhelmed with Chromium tabs rather than a terminal output. It constantly refreshes and pops up a new chromium tab making my Mac unusable. It also is only searching for the 3070 when I had it also searching for the 3080 and the ryzen 7 5800x. Attached bellow is a PDF of my .env file. 
[envfile.pdf](https://github.com/jef/streetmerchant/files/5639226/envfile.pdf)

**Comment by MercifulShadow** [comment]
You need to remove the spaces after the comma in the .env. It should look like this
3080,3070,ryzen5800
nvidia,amazon,newegg,bestbuy,bandh

**Comment by occasional-it-guy** [comment]
@MercifulShadow I did that however it's still using chromium to find only 3070's and from bestbuy, newegg, and amazon and it's still not utilizing the terminal like the github page shows. What you suggested helped fix somethings but not everything but thank you for helping with what you did!
